admin/bower.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "OSF_admin",
+  "dependencies": {
+    "jquery": "^1.11.2",
```

Comment:
@sloria Should I add in support for css? And do we push this into the the webpack somehow?